web-server/src/components/Teams/CreateTeams.tsx

```diff
@@ -312,58 +324,94 @@ const ActionTray: FC<CRUDProps> = ({
   );
 };
 
-const DisplayRepos: FC<{ hideCardComponents?: boolean }> = ({
-  hideCardComponents
-}) => {
+const DisplayRepos: FC = () => {
   const { selectedRepos } = useTeamCRUD();
+  const theme = useTheme();
+  if (!selectedRepos.length) return;
   return (
-    <FlexBox gap2 ml={2} minHeight={hideCardComponents ? '54px' : '49px'}>
-      {!!selectedRepos.length && <Divider flexItem orientation="vertical" />}
-      <FlexBox flexWrap={'wrap'} gap2>
-        {selectedRepos.map((repo) => (
-          <RepoItem
-            hideCardComponents={hideCardComponents}
-            repo={repo}
-            key={repo.id}
-          />
-        ))}
-      </FlexBox>
-    </FlexBox>
+    <TableContainer
+      sx={{
+        border: `2px solid ${theme.colors.secondary.light}`,
+        borderRadius: 1
+      }}
+    >
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell sx={{ px: 2 }}>Repo</TableCell>
+            <TableCell align="center" sx={{ p: 1 }}>
+              Deployed Via
+            </TableCell>
+            <TableCell align="center" sx={{ p: 1 }}>
+              Action
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {selectedRepos.map((repo) => (
+            <TableRow key={repo.id}>
+              <TableCell sx={{ px: 2 }}>{repo.name}</TableCell>
+              <TableCell align="center" sx={{ px: 2 }}>
```

Comment:
Same as the last one.

Reply:
fixed
![image](https://github.com/middlewarehq/middleware/assets/60030881/e5893e4c-003f-4660-b55e-8c2cb833c2f8)